python/ray/data/read_api.py

```diff
@@ -1755,6 +1761,109 @@ def create_connection():
     )
 
 
+@PublicAPI(stability="alpha")
+def read_databricks_uc_tables(
```

Comment:
Thoughts on `read_databricks` vs. `read_databricks_tables` vs. `read_databricks_uc_tables`? `read_databricks_uc_tables` sounds slightly verbose to me, but I'm not sure if it'd be misleading if we excluded "tables" or "uc".

Reply:
I think `read_databricks_tables` is good :)